release.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+echo "This script will stop if an unhandled error occurs";
+set -e -o pipefail
+
+
+read -p "Start the release process (y/n)?" choice
+case "${choice}" in
+  y|Y ) echo "";;
+  n|N ) exit;;
+  * ) echo "unknown response, exiting"; exit;;
+esac
+
+
+if  !  mvn -v | grep -q "Java version: 1.8."; then
+         echo "You need to use Java 8!";
```

Comment:
I don't think we need to be this strict, exactly. The maven compiler is set to target 1.8, so even if you use a newer version of Java, it should still produce Java 8-compatible bytecode.

Reply:
As always this is not completely true as the build currently fails when using Java 11, of course. Keep it in, we can change it once we have build running on java 11 :)